std/digest/package.d

```diff
@@ -1212,3 +1212,108 @@ if (isInputRange!R1 && isInputRange!R2 && !isInfinite!R1 && !isInfinite!R2 &&
         assert(!secureEqual(hex1, hex2));
     }
 }
+
+/**
+ * Used to convert hex string to byte array
```

Comment:
This needs to respect the ddoc format. Take a look at other documentation examples. Essentially, you need to have  `Parameters` and `Returns` sections.